modules/Core/pages/forgot_password.php

```diff
@@ -23,9 +23,21 @@
     // Enter email address form
     if (Input::exists()) {
         if (Token::check()) {
-            if (!isset($_POST['email']) || empty($_POST['email'])) {
-                $error = $language->get('user', 'email_required');
-            } else {
+
+            $rate_limit = [1, 60]; // 1 attempt every 60 seconds
+            $validation = Validate::check([
+                'email' => [
+                    Validate::REQUIRED => true,
+                    Validate::RATE_LIMIT => $rate_limit
```

Comment:
```suggestion
                    Validate::RATE_LIMIT => [1, 60]; // 1 attempt every 60 seconds
```

Reply:
Cant have a ; at the end as we're in an array. I'll see if I can do that tomorrow

Reply:
Ah typo